ci-operator/step-registry/cucushift/installer/wait/cucushift-installer-wait-commands.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 sleep "$SLEEP_DURATION" &
-wait
\ No newline at end of file
+wait
```

Comment:
@SachinNinganure  I think you can simply revert this file so that the PR doesn't need my approval